website/docs/guides/delivery.mdx

```diff
@@ -0,0 +1,473 @@
+metadata:
+  name: flagger
+```
+
+Then, to make the Flagger Helm repository available in the cluster, add the following manifest to the cluster repository:
+
+```yaml title="flagger/source.yaml"
+apiVersion: source.toolkit.fluxcd.io/v1beta2
+kind: HelmRepository
+metadata:
+  name: flagger
+spec:
+  interval: 1h
+  url: https://flagger.app
+```
+
+Then, to install the latest version of Flagger and the load tester app (which is used to generate traffic during the analysis phase), add the following `HelmRelease` manifests to the cluster repository:
+
+```yaml title="flagger/releases.yaml"
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: flagger
+spec:
+  releaseName: flagger
+  install:
+    crds: Create
+  upgrade:
+    crds: CreateReplace
+  interval: 10m
+  chart:
+    spec:
+      chart: flagger
+      reconcileStrategy: ChartVersion
+      sourceRef:
+        kind: HelmRepository
+        name: flagger
+  values:
+    metricsServer: http://prometheus.linkerd-viz:9090
+    meshProvider: linkerd
+---
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: loadtester
+spec:
+  interval: 10m
+  chart:
+    spec:
+      chart: loadtester
+      reconcileStrategy: ChartVersion
+      sourceRef:
+        kind: HelmRepository
+        name: flagger
+```
+
+In this case, Flagger is configured to work with the Linkerd service mesh. [Other providers](https://docs.flagger.app/install/flagger-install-on-kubernetes) may also be used instead of Linkerd to integrate with Flagger.
+
+Finally,  add the following Kustomization file that references all the previous files that were added:
+
+```yaml title="flagger/kustomization.yaml"
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: flagger
+resources:
+- namespace.yaml
+- source.yaml
+- releases.yaml
+```
+
+## Installing Linkerd using Flux
+
+For the Linkerd installation, a Kustomization file will be used.
+
+In order to support mTLS connections between meshed pods, Linkerd requires a trust anchor certificate and an issuer certificate with its corresponding key. These certificates are automatically created when the `linkerd install` command is used but when using a Helm chart to install Linkerd, these certificates need to be provided.
+
+Follow [these instructions](https://linkerd.io/2.11/tasks/generate-certificates/#generating-the-certificates-with-step) to generate the certificates using the `step` CLI.
+
+To generate the trust anchor certificate run:
+```bash
+step certificate create root.linkerd.cluster.local ca.crt ca.key \
+--profile root-ca --no-password --insecure
+```
+
+To generate the issuer certificate run:
+```bash
+step certificate create identity.linkerd.cluster.local issuer.crt issuer.key \
+--profile intermediate-ca --not-after 8760h --no-password --insecure \
+--ca ca.crt --ca-key ca.key
+```
+
+Add the `ca.crt`, `issuer.crt` and `issuer.key` files to the cluster repository under a `linkerd` directory.
```

Comment:
maybe mirror this approach for Flagger and suggest the creation of a Flagger directory? Then after each stage we could have a pseudo repo structure shown like

/clusters/my-cluster
|-linkerd
  |- files
|- flagger
  |- files
|- flux-system

wdyt?

Reply:
No, I think that would help the reader confirm that they have all the right files in place.